completion/bash/toolbox

```diff
@@ -10,11 +10,10 @@ __toolbox_images() {
 }
 
 __toolbox() {
-  local MIN_VERSION=29
-  local RAWHIDE_VERSION=31
+  local MIN_VERSION=28
```

Comment:
I think `MIN_VERSION` should continue to be 29 because Fedora 28 is now end-of-life. Thanks for updating `RAWHIDE_VERSION`, though.

Reply:
That's true. I didn't realize that.